### DISCUSSION TITLE
[Request] icon-variant for menu.item

### DISCUSSION THREAD

**Discussion by seabasss** [livewire/flux#473] [Feature Requests]
Any chance you can add icon-variant to menu.item? Looking to use outline, but only solid works.

Thanks for considering!

**Comment by clarkeash** [comment]
I just ran into the same issue, this would be really useful.
Maybe something like `icon-name:variant` or `icon-name-variant` could be useful to avoid needing an `iconVariant` prop all over

**Comment by thierroboto** [comment]
I agree as well, would be useful for buttons too.
